pkg/olog/audit.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 Outreach Corporation. All Rights Reserved.
+
+// Description: Contains a strongly-typed logger for audit logs using
+// slog.
+//
+// Note: This is an EXAMPLE of an extension to the slog.Logger, it is
+// not meant to actually be used.
```

Comment:
Important: This shows what an extension to the logger would look like if someone wanted to support logging structured logs for a specific thing (e.g., `audit` logs)